app/controllers/self_management_controller.rb

```diff
@@ -0,0 +1,70 @@
+class SelfManagementController < ApplicationController
+  def view_my_booking
+    @my_booking = current_user.bookings.order_by_time_desc.paginate(
+      page: params[:page], per_page: Settings.booking_per_page
+    )
+  end
+
+  def delete_my_booking
+    @my_booking = current_user.bookings.find_by_id params[:booking_id]
+    if @my_booking.destroy
+      respond_to do |format|
+        format.html{redirect_to view_my_booking_path}
+        format.js
+      end
+    else
+      flash[:danger] = t "controller.flash.delete_failed"
+      redirect_to view_my_booking_path
+    end
+  end
+
+  def view_my_review
```

Comment:
mấy method này toàn bộ đều là action hết ah cậu? 
Tìm hiểu lại restfult và nguyên nhân ra đời của restful nhé (luom)

Reply:
dạ tại em gộp mấy cái controller lại thành self_management luôn ạ. Xài restful như tutorial em vẫn có xài 2 method là edit với update (thông tin bản thân) ở action phía dưới ạ. Còn mấy cái trùng lặp em đặt tên action riêng ạ.

Reply:
Dạ action là trên private còn method dưới private ạ

Reply:
controller này ko sửa ah gái